### DISCUSSION TITLE
Enhancement to copy JTRACKGALLERY script for track to Clipboard

### DISCUSSION THREAD

**Discussion by noelada** [mastervanleeuwen/J-TrackGallery#53] [Ideas]
We have added a link in the Track Downloads area that when clicked copies the JTRACKGALLERY script to the clipboard:
![clipboardcopy1](https://user-images.githubusercontent.com/121601487/235825391-5690d1b3-1421-4a72-b388-0f88de8b62d3.png)
This enhancement makes it easy to add the track to the place it is to be displayed (simply by pasting the clipboard). An example of a pasted script is **{JTRACKGALLERYMAP} gpxfilename=walk-20210131-mtriddell.gpx, show_link=1 {/JTRACKGALLERYMAP}**.
Perhaps Marco you could include this option in a future release as I believe other users would find it helpful.
Noel Adams.

**Comment by mastervanleeuwen** [comment]
This has been implemented; it is a configurable option for the track view.

**Comment by noelada** [comment]
Thanks.
Noel.
________________________________
